worker/worker.go

```diff
@@ -331,6 +332,16 @@ func (w *Worker) getExtraIssues(name, owner *githubv4.String, before *string, re
 
 }
 
+// Parse string to time.Time
+func timeConvert(GHUpdatedTime string) time.Time {
+	layout := "2006-01-02T15:04:05"
+	t, err := time.Parse(layout, GHUpdatedTime)
+	if err != nil {
+	    fmt.Println(err)
```

Comment:
Could you handle the error here? return empty time or something

Reply:
`time.Parse()` will return error on malformed input and the error will be printed. In the case of an error variable `t` is initialized to zero value of `time.Time`(i.e. `0001-01-01 00:00:00 +0000 UTC`) and we are returning `t` on Line342. So there should not be a need to write another return inside the if statement.

Reply:
you are right, sorry 

I see that github's response is "updatedAt": "2018-11-14T00:08:47Z" shouldn’t there be a Z at the end of your layout?

Reply:
The updatedAt field returns `An ISO-8601 encoded UTC date string` so I thought as timezone will not be returned, but the Z is always appended to signify UTC. 
Thanks for pointing it out.

Reply:
If I'm not mistaken ISO-8601 should be compatible with time.RFC3339, could you use time.RFC3339 here?

Reply:
Okay, made the changes.